src/engine/guidance/post_processing.cpp

```diff
@@ -27,6 +27,29 @@ namespace guidance
 
 namespace detail
 {
+
+void print(const std::vector<RouteStep> &steps)
+{
+    std::cout << "Path\n";
+    int segment = 0;
+    for (const auto &step : steps)
+    {
+        const auto type = static_cast<int>(step.maneuver.instruction.type);
```

Comment:
You can use std::underlying_type to make the code work independently of your enum's representation